### DISCUSSION TITLE
Password failed message

### DISCUSSION THREAD

**Discussion by Steffen-99** [laravel/framework#40032] [Q&A]
Hi,

how its possible to get translated this messages:

https://github.com/laravel/framework/blob/07c14c52de0783a583e2bb7233d31d8e10fad036/src/Illuminate/Validation/Rules/Password.php#L284

**Comment by JurianArie** [comment]
You can translate it by creating a json file for your locale. 

Docs: https://laravel.com/docs/8.x/localization#using-translation-strings-as-keys

Example: https://github.com/Laravel-Lang/lang/blob/f9cbabc0946bcf431494173c1505ba3285b657a0/locales/nl/nl.json#L578
